common/src/main/java/com/gregtechceu/gtceu/common/machine/storage/QuantumChestMachine.java

```diff
@@ -74,7 +77,7 @@ public class QuantumChestMachine extends TieredMachine implements IAutoOutputIte
     protected ISubscription exportItemSubs;
     @Persisted @Getter @Setter
     private boolean isVoiding;
-    @Persisted @Getter
+    @Persisted @DescSynced @Getter
```

Comment:
Should this maybe also have `@RequireRenderer`? Unsure

Reply:
The renderer runs on every tick anyway, so I've decided to use the same annotations we had on `stored` for this as well.

Reply:
Ok cool